openstack/compute/v2/servers/testing/fixtures.go

```diff
@@ -870,17 +870,17 @@ var ListAddressesExpected = map[string][]servers.Address{
 	"public": []servers.Address{
 		{
 			Version: 4,
-			Address: "80.56.136.39",
+			Address: "50.56.176.35",
```

Comment:
Are all of these small changes required to make the unit tests pass (i.e. these unit tests have always been broken)?